blog/2024-09-10-pants-2-22/index.mdx

```diff
@@ -16,7 +16,7 @@
 
 {/* truncate */}
 
-We are pleased to announce Pants [2.22.0](<(https://github.com/pantsbuild/pants/tree/2.22.x/src/python/pants/notes/2.22.x.md)>), the latest release of [Pantsbuild, the scalable and ergonomic build system](https://www.pantsbuild.org/). To update, set `pants_version = "2.22.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/2.22/docs/getting-started).
+We are pleased to announce Pants [2.22.0](<(https://github.com/pantsbuild/pants/tree/2.22.x/docs/notes/2.22.x.md)>), the latest release of [Pantsbuild, the scalable and ergonomic build system](https://www.pantsbuild.org/). To update, set `pants_version = "2.22.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/2.22/docs/getting-started).
```

Comment:
In addition to the wrong path, It looks like the first link here has some extra syntax that breaks the link:

```suggestion
We are pleased to announce Pants [2.22.0](https://github.com/pantsbuild/pants/tree/2.22.x/docs/notes/2.22.x.md), the latest release of [Pantsbuild, the scalable and ergonomic build system](https://www.pantsbuild.org/). To update, set `pants_version = "2.22.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/2.22/docs/getting-started).
```